src/main/webapp/Portal.html

```diff
@@ -159,382 +159,70 @@
+      loadJs(cdnEndpoint + 'generated/jquery.min.js')
+      loadJs('//ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js')
+      loadJs(cdnEndpoint + 'js/back-forward-nav-handler.js')
+      loadJs(cdnEndpoint + 'generated/moment.min.js')
+      loadJs(cdnEndpoint + 'generated/jsplumb.min.js')
+      loadJs(cdnEndpoint + 'js/jquery.tablesorter.min.js')
+      loadJs(cdnEndpoint + 'js/jquery.tablesorter.scinot.js')
+      loadCss(cdnEndpoint + 'icons/font/synapseicons.css?v=1')
+      loadCss(cdnEndpoint + 'generated/font-awesome.css')
+      loadJs(cdnEndpoint + 'js/highlight.pack.js')
+      loadJs(cdnEndpoint + 'js/init-nchart.js')
+      loadJs(cdnEndpoint + 'js/markdown-it.min.js')
+      loadJs(cdnEndpoint + 'js/markdown-it-plugins-18.min.js')
+      loadJs(cdnEndpoint + 'js/polyfill-2.js')
+      loadJs(cdnEndpoint + 'js/details-shim.min.js')
+      loadJs(cdnEndpoint + 'js/xss.min.js')
+      loadJs(cdnEndpoint + 'generated/pica.min.js')
+      loadJs(reactPath)
+      loadJs(reactDomPath)
+      loadJs(cdnEndpoint + 'generated/react-transition-group.min.js')
+      loadJs(cdnEndpoint + 'generated/prop-types.min.js')
+      loadJs(cdnEndpoint + 'generated/react-measure.js')
+      loadJs(cdnEndpoint + 'generated/plotly.min.js')
+      loadJs(cdnEndpoint + 'generated/create-plotly-component.min.js')
+      loadJs(cdnEndpoint + 'generated/react-router-dom.min.js')
+      loadJs(cdnEndpoint + 'generated/universalCookie.min.js')
+      loadJs(cdnEndpoint + 'generated/react-bootstrap.min.js')
+      loadJs(srcPath) // We don't load SRC's scss because we include it in our own compiled scss
+      loadJs(cdnEndpoint + 'generated/croppie.min.js')
+      loadCss(cdnEndpoint + 'js/diff/diffview.css')
+      loadCss(cdnEndpoint + 'css/details-shim.min.css')
+      loadCss(cdnEndpoint + 'generated/croppie.css')
+      loadCss(cdnEndpoint + 'css/highlight-js-default.css')
+      loadCss(
         '//ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/base/jquery-ui.css',
       )
-      head.load(cdnEndpoint + 'js/linkify.min.js')
-      head.load(cdnEndpoint + 'js/linkify-string.min.js')
-      head.load(cdnEndpoint + 'css/katex-0.10.1.min.css')
-      head.load('//cdn.statuspage.io/se-v2.js')
-      // head.load("//kh896k90gyvg.statuspage.io/embed/script.js");
-      head.load(cdnEndpoint + 'generated/papaparse.min.js')
+      loadJs(cdnEndpoint + 'js/linkify.min.js')
+      loadJs(cdnEndpoint + 'js/linkify-string.min.js')
+      loadCss(cdnEndpoint + 'css/katex-0.10.1.min.css')
+      loadJs('//cdn.statuspage.io/se-v2.js')
+      // loadJs("//kh896k90gyvg.statuspage.io/embed/script.js");
+      loadJs(cdnEndpoint + 'generated/papaparse.min.js')
+    </script>
+    <script type="module">
+      // react-query only ships ESM. esm.sh is a CDN that will 'bundle' dependencies for us
+      import * as ReactQuery from 'https://esm.sh/@tanstack/react-query@5.22.2'
+      window.ReactQuery = ReactQuery
     </script>
```

Comment:
Load ReactQuery from the CDN, and store it in the `window.ReactQuery` variable just as the UMD version did.